src/codegen/BUILD

```diff
@@ -127,3 +131,22 @@ cc_library(
     ],
 )
 
+py_binary(
+  name = "static_def_gen",
+  srcs = ["static_def_gen.py"],
+)
+
+genrule(
```

Comment:
A comment explaining this would be beneficial

Reply:
Done.